akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2Multiplexer.scala

```diff
@@ -248,7 +248,7 @@ private[http2] trait Http2MultiplexerSupport { logic: GraphStageLogic with Stage
       // WaitingForNetworkToSendData: Data frames queued but no network demand
       // WaitingForConnectionWindow: Data frames queued, demand from the network, but no connection-level window available
 
-      case object Idle extends MultiplexerState {
+      private[http2] case object Idle extends MultiplexerState {
```

Comment:
To help the Scala compiler notice that nobody will see this class outside `http2`. It complained about:
> method pushControlFrame in class WaitingForConnectionWindow references private[http2] trait FrameEvent.